.github/workflows/test-chart.yml

```diff
@@ -39,25 +39,55 @@ jobs:
         run: |
           make docker-build
 
-      - name: Install Helm
+      - name: Install helm
```

Comment:
I believe explicit installation isn't required as per https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md, I suppose kubectl is also being used from existing env (or installed during minikube)?

Nevertheless, I think it's better to reuse whatever is available if they are in required version range?

Reply:
using version explicitly to avoid dependency problem on the base image.

Reply:
And also we will test with latest releases from helm